agent/consul/rpc_test.go

```diff
@@ -142,6 +142,50 @@ func TestRPC_NoLeader_Retry(t *testing.T) {
 	}
 }
 
+func TestRPC_getLeader_ErrLeaderNotTracked(t *testing.T) {
```

Comment:
This test could use a skip block since it's runtime is > 100ms

```go
if testing.Short() {
	t.Skip("too slow for testing.Short")
}
```

Reply:
Done.